Mathmagician.Tests/FibonacciTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class FibonacciTest
+    {
+        [TestMethod]
+         public void FibEnsureICanCreateAnInstance()
+        {
+            //Arrange
+
+            //Act
+            Fibonacci my_fib = new Fibonacci();//What I want to be able to do
+
+            //Assert
+            Assert.IsNotNull(my_fib);
+        }
+
+        [TestMethod]
+        public void FibNotNullTest()
+        {
+            Fibonacci my_fib = new Fibonacci();
+            my_fib.GenerateFibonacci(5);
+            int temp = my_fib.FibonacciList[0];
+
+            //Assert
+            Assert.IsNotNull(temp);
```

Comment:
@dupender This test isn't useful. Maybe you should check for the first Fibonacci?